components/footer.js

```diff
@@ -1,16 +1,19 @@
 export default () => (
-    <footer className="footer">
-        <div className="container">
-            <div className="content has-text-centered" id="download">
-                <p>
-                    <strong>Buttercup</strong> is made by <a href="https://twitter.com/perry_mitchell" rel="noopener" target="_blank">Perry Mitchell</a> & <a href="https://twitter.com/sallar" rel="noopener" target="_blank">Sallar Kaboli</a> in Helsinki, Finland.
-                    </p>
-                <p>
-                    <a className="icon" href="https://github.com/buttercup/buttercup-desktop" rel="noopener" target="_blank">
-                        <i className="fa fa-github"></i>
-                    </a>
-                </p>
-            </div>
-        </div>
-    </footer>
+  <footer className="footer">
+    <div className="container">
+      <div className="content has-text-centered">
+        <p>
+          <strong>Buttercup</strong> is made by{' '}
```

Comment:
Sorry to be pedantic, but do you prefer these over `&nbsp;`?

Reply:
@perry-mitchell the issue here is, react removes all whitespace between elements, so if you wanna put the <a> tag on next line it will stick to previous element 😐 `&nbsp;` would be a good idea but that's actually a non-breaking space so it doesn't break on smaller screen (doesn't wrap).

Reply:
Fair enough. This is unfortunately a bit ugly imho, but I can see the issue it addresses. Doesn't seem to be any helpful way around this either - I might have expected a babel helper to deal with this perhaps, but nope 😬.

Reply:
@perry-mitchell oh you dont have to tell me its VERY ugly imo. Hopefully we can find a better solution to this